Sitemap/Url/GoogleNewsUrlDecorator.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Presta\SitemapBundle\Sitemap\Url;
 
+use DateTime;
```

Comment:
Symfony usually does not import classes from global namespace

Reply:
Since it not in any pratice guide, I assume this is matter of preference.